SPECIFICATIONS.md

```diff
@@ -694,20 +696,59 @@ message Biscuit {
   repeated SignedBlock blocks = 3;
   required Proof proof = 4;
 }
+```
+
+The `rootKeyId` is a hint to decide which root public key should be used
+for signature verification.
+The `authority` block has a special meaning, as it is signed by the root key,
+so it is stored separately.
+The `blocks` field contains the rest of the block list, that must stay in the same order.
+The `proof` field contains either:
+- `nextSecret`: the private key corresponding to the `nextKey` public key in the last block (attenuable tokens)
+- `finalSignature`: a signature of the last block by that private key (sealed tokens).
+
+```proto
+message Proof {
+  oneof Content {
+    bytes nextSecret = 1;
+    bytes finalSignature = 2;
+  }
+}
+```
 
+The `SignedBlock` structure carries the sigend data and its signatures, representing the
+chain of trust starting from the root key.
+
+```proto
 message SignedBlock {
   required bytes block = 1;
   required PublicKey nextKey = 2;
   required bytes signature = 3;
   optional ExternalSignature externalSignature = 4;
   optional uint32 version = 5;
 }
+```
+
+Each `SignedBlock` contains:
+- `block`: a `Block` structure serialized as a byte array
```

Comment:
maybe we should hint that why there are two layers of serialization (unless it’s already done somewhere else)